[PR ENHANCEMENT] - View Individual User view the Admin Panel

NicholasWahome opened the pull request:
Fixed Indivual user view for the admin panel linking.

PR Resolves issue: #438

NicholasWahome committed:
View Individual User view the Admin Panel

NicholasWahome committed:
Prettified Code!

NicholasWahome committed:
Merge branch 'main' into adminViewLinking

NicholasWahome committed:
Prettified Code!

NicholasWahome committed:
Merge branch 'main' into adminViewLinking

harismah reviewed (changes requested):
- Remove header and footer for when viewing a user profile, it should look like this with the back option.
![image](https://github.com/user-attachments/assets/37b4e105-fc8d-4cbc-b149-8f4498d5eb50)

v-vasco committed:
Merge branch 'main' into adminViewLinking

v-vasco committed:
Update AdminUsers.jsx

Small change to show the selected user's profile in the AdminUsers page without leaving the page and without displaying the header or footer

v-vasco committed:
Prettified Code!

v-vasco committed:
Update SeeProfile.jsx

Updated to work with AdminUsers page without interfering with its regular use

v-vasco committed:
Prettified Code!

v-vasco committed:
Update AdminUsers.module.css

v-vasco committed:
Prettified Code!

v-vasco committed:
Update SeeProfile.jsx

Fixed style issue

v-vasco committed:
Prettified Code!

v-vasco commented:
I've applied the requested changes to the AdminUsers page as shown in the demo below.

https://github.com/user-attachments/assets/bbc08a1d-d13c-4712-a45d-5e14020c7d94

I modified the SeeProfile page slightly so it could be reused in the AdminUsers page. Below is a demo showing the SeeProfile page still works normally on the non-admin frontend.

https://github.com/user-attachments/assets/1a891c8c-d2ff-4e66-88f1-7987909f347a

v-vasco committed:
Update AdminUsers.jsx

v-vasco committed:
Prettified Code!

v-vasco committed:
Update SeeProfile.jsx

v-vasco committed:
Prettified Code!

v-vasco committed:
Merge branch 'main' into adminViewLinking

v-vasco committed:
Merge branch 'main' into adminViewLinking

v-vasco committed:
Merge branch 'main' into adminViewLinking

harismah reviewed (approved):
Approved, everything is working as intended. Great work!
